src/main/resources/db/migration/V1_34__update_region_mapping.sql

```diff
@@ -0,0 +1,36 @@
+truncate table  jrd_lrd_region_mapping  cascade;
```

Comment:
I think cascade will remove data from parent table as well. Is this expected?
FYI @kavithagade12

Reply:
this is standalone table and do not have relations with other table i.e no primary key foreign key . 
cascade do not have impact . we can remove it as well .